docs/UserGuide.md

```diff
@@ -303,110 +374,121 @@ Format: `clear`
+| GUI               | Graphical User Interface: A visual way for users to interact with the application through graphical elements such as buttons and icons. |
+| JSON              | JavaScript Object Notation: A format for structuring data, used here for saving and loading user data from a file.                      |
+| Membership Tiers  | Designated levels within the loyalty program offering different benefits, determined by the amount of points a member has accumulated.  |
+| Seed Data         | Preloaded data used to demonstrate the application's capabilities without needing to enter new data manually.                           |
+| Allergen          | Allergens that customers have, for the baker to take note when making an order.                                                         |
+| GF                | Gluten-free: An allergen specification.                                                                                                 |
+| LF                | Lactose-free: An allergen specification.                                                                                                |
+| Order Details     | Information related to purchases made by a member, used for tracking the orders made by the customer.                                   |
+| Points            | A system of credit awarded to members for their purchases, contributing to their membership tier placement.                             |
+| Jar File          | A compressed format for distributing bundled Java applications and libraries.                                                           |
 
 --------------------------------------------------------------------------------------------------------------------
 
-## FAQ
+# 6. FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**A**: Install SweetRewards on the other computer and copy the data file from your original installation directory to the new one. This will replace the empty data file created during the new installation.
+
+**Q**: How do I manually add points to a member's account?<br>
+**A**: Use the addpoints command with the member's name and the desired amount of points. For instance: addpoints n/John Doe p/50 adds 50 points to John Doe's account, useful for promotional points or correcting errors.
 
-**Q**: How can I add points to a member's account?
-**A**: Use the addpoints command with the member's name and the number of points you want to add. For example, addpoints n/John Doe p/50 adds 50 points to John Doe’s account.
+**Q**: How can I upgrade a member's tier manually?<br>
+**A**: Employ the addmship command with the member's name and the intended new tier. For example, addmship n/John Doe ms/Gold manually upgrades John Doe to the Gold tier, ideal for special recognitions.
```

Comment:
I think this is outdated, only can add points so:

Employ the addmempts command with the member's name and the points you want to add. For example, addmempts n/John Doe mp/400 manually awards John Doe 400 points. The membership tier of the member will be determined by the number of points they have according to the **membership tier table** <----- (Add a hyperlink to table here)